sources/web/datalab/polymer/components/item-list/item-list.ts

```diff
@@ -54,6 +57,16 @@ class ItemListElement extends Polymer.Element {
    */
   public columns: Array<string>;
 
+  /**
+   * Whether to hide the header row
+   */
+  public noHeader: boolean;
```

Comment:
Yes, but the usage will then have to be `<item-list show-header="false" enable-selection="false>` rather than simply `<item-list hide-header disable-selection>`, which is a much smoother design.
I think using the negative name is fine since it's not the main experience.